model/bin/ww3_from_ftp.sh

```diff
@@ -10,19 +10,32 @@ curr_dir=`pwd`
 # Set WW3 code version
 ww3ver=v7.12.6
 
+interactive='y'
+if [ $# -eq 3 ] ; then
+  interactive='n'
+  ww3dir=$1  # [ '../../']
+  wnew=$2    # ['y']
+  wnew2=$3   # ['y']
+fi
+
```

Comment:
Does it solve https://github.com/NOAA-EMC/WW3/issues/578? if yes, it should be a separate PR.